modules/git/object_format.go

```diff
@@ -5,22 +5,15 @@ package git
 
 import (
 	"crypto/sha1"
-	"fmt"
 	"regexp"
-	"strings"
-)
-
-type ObjectFormatID int
-
-const (
-	Sha1 ObjectFormatID = iota
 )
 
 // sha1Pattern can be used to determine if a string is an valid sha
 var sha1Pattern = regexp.MustCompile(`^[0-9a-f]{4,40}$`)
 
+const Sha1ObjectFormatName = "sha1"
```

Comment:
I think it could be:

```
type ObjectFormat string

const ObjectFormatSha1 ObjectFormat = "sha1"
```

or 

```
type ObjectFormat interface { } 

const ObjectFormatSha1 = ObjectFormatSha1Impl{}

repo.ObjectFormatName = ObjectFormatSha1.Name()
```

Will take a look later.

Reply:
Well, the 2nd option very least because we have methods specific to each objectFormat and this abstraction is used throughout the codebase.

But this doesn't need to be changed here since it's just a temporary hack that will be addressed in SHA256 patch.

Reply:
Followed 2nd option.